gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -52,8 +52,8 @@
     [10131, "dround", "BIGINT", ["DOUBLE"], "MathFunctions::round"],
     [10132, "round", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::round_up_to"],
     [10133, "dround", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::round_up_to"],
-    [10134, "truncate", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::truncate"],
-    [10135, "truncate_decimal", "DECIMAL128", ["DECIMAL128", "INT"], "MathFunctions::truncate_decimal128"],
+    [10134, "truncate", "DECIMAL128", ["DECIMAL128", "INT"], "MathFunctions::truncate_decimal128"],
+    [10135, "truncate", "DOUBLE", ["DOUBLE", "INT"], "MathFunctions::truncate"],
```

Comment:
The modify will case compatibility problem when be was updated and fe didn't update

Reply:
Ok, let me figure it out

Reply:
fixed